pages/edge/learn/connecting-domains-to-edge.mdx

```diff
@@ -0,0 +1,65 @@
+import { Callout } from "nextra-theme-docs";
+
+# Connecting your domain to Wasmer Edge
+
+## Introduction
+
+For any functional website/server, you need a domain name. This is the address that people will use to access your services.
+For example, the domain name for this website is `wasmer.io` and the domain for our registry is `registry.wasmer.io`.
+
+In this guide, we will show you how to connect your domain to Wasmer Edge.
+
+## Subdomain
+
+If you want to connect a subdomain(e.g. `blogs.wasmer.io`) to your app hosted on Wasmer Edge, you will need to create a CNAME record pointing to your app on wasmer edge.
+
+```yaml
+blogs.wasmer.io. 0 3600 IN CNAME wasmer-blogs.wasmer.app.
+```
+
+And it should be done! You can now access your app on `blogs.wasmer.io`.
+
+## Root domain
+
+Connecting a root domain is a bit more complicated than connecting a subdomain because you can't create a CNAME record for a root domain. But there are some providers that allow you to create a CNAME record for a root domain. If your provider doesn't allow you to do that, you can use a service like [Cloudflare](https://www.cloudflare.com/) to create a CNAME record for your root domain.
```

Comment:
```suggestion
Connecting a root domain is a bit more complicated than connecting a subdomain, because you can't create a CNAME record for a root domain. 

Some large DNS providers emulate root CNAME records. If your provider doesn't allow you to do that, you can use a service like [Cloudflare](https://www.cloudflare.com/).
Note: this requires moving your entire DNS setup to Clouflare.
```